src/components/Layout/SlideMenu/SlideMenu.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import { slide as Menu } from 'react-burger-menu';
+import '../../../stylesheets/components/SlideMenu.scss';
+
+class SlideMenu extends React.PureComponent {
+  render() {
+    return (
+      <Menu right>
+        <Link to="/">Programs</Link>
+        <li><Link to="/annual-report-2018">Impact</Link></li>
```

Comment:
Is there a reason that this is the only menu item that is wrapped in a `<li>` tag in this menu? Also, is there a reason that this hamburger menu mostly does not use `<li>` tags even though the non-hamburger header and the footer use `<li>` tags?

Reply:
no reason... this was just my mistake